c# generating script code'

silverspoon19931120 opened the pull request:
<!-- Thanks for submitting a pull request! We appreciate you spending the time to work on these changes. Please provide enough information so that others can review your pull request. These fields below are mandatory. -->

## Summary

<!-- Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request. If it fixes a bug or resolves a feature request, be sure to link to that issue. -->

## Types of changes

_What types of changes does your code introduce to this project? Put an `x` in the boxes that apply._

- [ ] Bugfix.
- [ ] New feature.
- [ ] Documentation update.

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the pull request. If you are unsure about any of them, do not hesitate to ask. We are here to help! This is simply a reminder of what we are going to look for before merging your code._

- [ ] I have read the [CONTRIBUTING](../CONTRIBUTING.md) documentation.
- [ ] I have rebased, and my remote branch is now up-to-date with the `master` branch of the upstream.
- [ ] Lint and unit tests pass locally with my changes.
- [ ] I have added tests that prove my changes are effective.
- [ ] I have added the necessary documentation (if appropriate).

## Further comments

<!-- If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, ... -->

ledinhphuong commented:
Please fix lint issues fist, more detail at https://github.com/kobiton/appium-script-generator/actions/runs/7005335910/job/19076430635?pr=6

ledinhphuong commented on .gitmodules:
```diff
@@ -1,4 +1,4 @@
 	path = appium-script-schema
 	branch = master
-	url = git@github.com:kobiton/appium-script-schema.git
+	url = https://github.com/kobiton/appium-script-schema.git
```
Could you please `rebase` your code base again I remembered that this change I added this [PR](https://github.com/kobiton/appium-script-generator/pull/4/files) last month.

silverspoon19931120 replied:
What do you mean?

ledinhphuong commented on CONTRIBUTING.md:
```diff
@@ -6,6 +6,7 @@
 |
 <strong><a href="#debugging-code">Debugging code</a></strong>
 </p>
+
```
This is the same concern with my above comment in `.gitmodules` file.

ledinhphuong commented on __tests__/integration/generate-script.test.js:
```diff
@@ -11,6 +11,8 @@ const SERVICE_PROTO_FILE = path.resolve(
   __dirname, '../../src/appium-script-schema/generate-script.proto')
 const JAVA_JUNIT_INPUT_FILE = path.resolve(
   __dirname, '../resource/java-junit-input.json')
+  const CSHARP_NUNIT_INPUT_FILE = path.resolve(
+    __dirname, '../resource/csharp-nunit-input.json')
```
Please correct the indentation.

ledinhphuong commented on __tests__/resource/csharp-nunit-input.json:
```diff
@@ -0,0 +1,251 @@
+    "apiUrl": "https://api.kobiton.com",
+    "portalUrl": "https://portal.kobiton.com",
+    "username": "kobitonadmin+appiumScriptGen",
+    "apiKey": "65530082-3eaa-44f8-adc5-e1d6d6a5a00c"
```
```suggestion
    "apiKey": "your_kobiton_api_key"
```

ledinhphuong commented on package.json:
```diff
@@ -6,8 +6,8 @@
     "start": "bash ./scripts/register-grpc-server.sh; nodemon src/index.js --exec babel-node",
-    "test": "jest",
-    "lint": "eslint src",
+    "test": "jest --forceExit",
+    "lint": "eslint \"src/**/*.js\"",
```
The same concern as above

silverspoon19931120 commented:
I fixed other issues,
But what do you mean this?
[image: image.png]

On Tue, Nov 28, 2023 at 6:53 AM Phuong Le ***@***.***> wrote:

> ***@***.**** requested changes on this pull request.
> ------------------------------
>
> In .gitmodules
> <https://github.com/kobiton/appium-script-generator/pull/6#discussion_r1407205469>
> :
>
> > @@ -1,4 +1,4 @@
>  [submodule "appium-script-schema"]
>  	path = appium-script-schema
>  	branch = master
> -	url = ***@***.***:kobiton/appium-script-schema.git
> +	url = https://github.com/kobiton/appium-script-schema.git
>
> Could you please rebase your code base again I remembered that this
> change I added this PR
> <https://github.com/kobiton/appium-script-generator/pull/4/files> last
> month.
> ------------------------------
>
> In CONTRIBUTING.md
> <https://github.com/kobiton/appium-script-generator/pull/6#discussion_r1407206493>
> :
>
> > @@ -6,6 +6,7 @@
>  |
>  <strong><a href="#debugging-code">Debugging code</a></strong>
>  </p>
> +
>
> This is the same concern with my above comment in .gitmodules file.
> ------------------------------
>
> In __tests__/integration/generate-script.test.js
> <https://github.com/kobiton/appium-script-generator/pull/6#discussion_r1407207402>
> :
>
> > +  const CSHARP_NUNIT_INPUT_FILE = path.resolve(
> +    __dirname, '../resource/csharp-nunit-input.json')
>
> Please correct the indentation.
> ------------------------------
>
> In __tests__/resource/csharp-nunit-input.json
> <https://github.com/kobiton/appium-script-generator/pull/6#discussion_r1407209638>
> :
>
> > @@ -0,0 +1,251 @@
> +{
> +  "sentAt": {
> +    "low": 1105186912,
> +    "high": 395,
> +    "unsigned": true
> +  },
> +  "serverInfo": {
> +    "apiUrl": "https://api.kobiton.com",
> +    "portalUrl": "https://portal.kobiton.com",
> +    "username": "kobitonadmin+appiumScriptGen",
> +    "apiKey": "65530082-3eaa-44f8-adc5-e1d6d6a5a00c"
>
> ⬇️ Suggested change
>
> -    "apiKey": "65530082-3eaa-44f8-adc5-e1d6d6a5a00c"
> +    "apiKey": "your_kobiton_api_key"
>
> ------------------------------
>
> In package.json
> <https://github.com/kobiton/appium-script-generator/pull/6#discussion_r1407215526>
> :
>
> > +    "test": "jest --forceExit",
> +    "lint": "eslint \"src/**/*.js\"",
>
> The same concern as above
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/kobiton/appium-script-generator/pull/6#pullrequestreview-1752061153>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/A6DMWLIUWZ44MQHCWWKPZBDYGV36ZAVCNFSM6AAAAAA737VYOOVHI2DSMVQWIX3LMV43YUDVNRWFEZLROVSXG5CSMV3GSZLXHMYTONJSGA3DCMJVGM>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>
